dev-libs/libgrapheme/libgrapheme-2.0.0.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A unicode string library"
+DESCRIPTION="libgrapheme is an extremely simple freestanding C99 library providing \
+utilities for properly handling strings according to the latest Unicode standard \
+15.0.0. It offers fully Unicode compliant."
+HOMEPAGE="https://libs.suckless.org/libgrapheme"
+COPYRIGHT="2022 Laslo Hunhold"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="https://dl.suckless.org/libgrapheme/libgrapheme-$portVersion.tar.gz"
+SOURCE_FILENAME="libgrapheme-$portVersion.tar.gz"
+CHECKSUM_SHA256="a6e1f7aec86af029dc183e32ab7a8d531fc00e54c44921dbdffc929aceb90dc6"
+PATCHES="libgrapheme-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libgrapheme$secondaryArchSuffix = $portVersion
+	lib:libgrapheme$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libgrapheme${secondaryArchSuffix}_devel = $portVersion
+	devel:libgrapheme$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	libgrapheme$secondaryArchSuffix == $portVersion
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	make install LIBPREFIX=$libDir INCPREFIX=$includeDir \
+		MANPREFIX=$manDir $jobArgs
+
+	packageEntries devel \
+		$developDir
+
+	rm -rf  $libDir/libgrapheme.a
```

Comment:
The `.a` file gets moved before `packageEntries devel` if you use `prepareInstalledDevelLib`, so the file is still around, move it before `prepareInstalledDevelLib` (some just keep it in, only the `.la` files are deleted from most recipes)

Reply:
updated and removed the .a entry removal